gateway-release/home/conf/gateway-site.xml

```diff
@@ -208,4 +208,10 @@ limitations under the License.
         <description>Add service name to x-forward-context header for the list of services defined above.</description>
     </property>
 
+    <!-- Async support required for SSE -->
```

Comment:
Yes its false by default. I removed it.
I thought since we have flags for other features (websocket) that are false be default it would be nice that add this one as well.